x-pack/plugins/ml/server/routes/schemas/saved_objects.ts

```diff
@@ -22,3 +22,8 @@ export const syncJobObjects = schema.object({ simulate: schema.maybe(schema.bool
 export const jobTypeSchema = schema.object({
   jobType: schema.string(),
 });
+
+export const canDeleteJobScheme = schema.object({
```

Comment:
Nit: This should be `canDeleteJobSchema`

Reply:
ugh

Reply:
well done for finding the deliberate mistake, fixed in https://github.com/elastic/kibana/pull/86436/commits/9fa1c29b6577d8edb8d4a513efb94df232b34676